include/chemfiles/Topology.hpp

```diff
@@ -126,6 +127,18 @@ class CHFL_EXPORT Topology final {
     /// @throws OutOfBounds if `atom_i` or `atom_j` are greater than `size()`
     void remove_bond(size_t atom_i, size_t atom_j);
 
+    /// Get the bond order for the given bond
+    ///
+    /// If the bond does not exist, this will return a out_of_bonds error
```

Comment:
out_of_bonds => out_of_bounds

Also, do we want to throw an error or to return BondOrder::UNKNOWN?

Reply:
Yeah, that's up for discussion.  I felt an error would be more appropriate and `UNKNOWN` really means unspecified. We could add something to the enum like `UNSPECIFIED` and use `UNKNOWN` for error cases. Thoughts?

Reply:
This sounds good! Is there a difference in some formats between unknown and unspecified ?

Reply:
I don't think so.  Some formats (PDB for example) just define everything as single bonds thus all bonds would be `UNSPECIFIED` because the format does not support bond orders.

Mol2 defines an `unknown` bond and a `dummy` bond for cases where the bond order cannot be determined using SYBYL typing rules.

Reply:
Thinking a bit more about this, having both `UNKNOWN` and `UNSPECIFIED` can be confusing for the users, about what the difference is between the two ...

We might want to use an exception for now, and experiment a bit with the API

Reply:
Ok, I've added some logic to MOL2 and SDF to use those format's way of interpreting an unknown bond as unspecified.

Reply:
This still contains the typo: out_of_bonds => out_of_bounds